prose/config.go

```diff
@@ -17,6 +17,7 @@ func NewConfigSite() *shared.ConfigSite {
 	dbURL := utils.GetEnv("DATABASE_URL", "")
 	maxSize := uint64(500 * utils.MB)
 	maxImgSize := int64(10 * utils.MB)
+	maxSpecialFileSize := int64(5 * utils.KB)
```

Comment:
Is there any reason in particular why you added this here in the prose service?

Reply:
Oh ehh I guess it's not necessary since the resulting ConfigSite struct will get a default of zero which is unused anyway, but I figured I should update all users of ConfigSite if I'm adding a new field. Is there a reason not to?

Reply:
I don't think it's necessary.  In fact, we are going to be slowly separating pgs from the rest of our services.